templates/repo/issue/view_content/comments.tmpl

```diff
@@ -795,6 +795,18 @@
 					{{else}}{{$.locale.Tr "repo.pulls.auto_merge_canceled_schedule_comment" $createdStr | Safe}}{{end}}
 				</span>
 			</div>
+		{{else if eq .Type 36}}
+			{{$link := printf "%s/commit/%s" $.Repository.HTMLURL ($.Issue.PullRequest.MergedCommitID|PathEscape)}}
```

Comment:
The link should be the tag's link but not the commit

Reply:
I think we can better keep the link to the commit here instead of the tag, because it's tagged to that specific commit on the time of tagging. 

If a user deletes the tag and retags later, the history of the tagging is still available in the timeline.